prow/config.yaml

```diff
@@ -5979,6 +5979,21 @@ presubmits:
         name: auto-generated-docker-graph-volume-mount
     trigger: (?m)^/test pull-security-kubernetes-verify-prow,?(\s+|$)
   kubernetes-sigs/cluster-api:
+  - name: pull-cluster-api-all
+    agent: kubernetes
+    always_run: true
+    context: pull-cluster-api-all
```

Comment:
Please add the work "make" in all places that this name is defined:

pull-cluster-api-make-all

Reply:
Another option is to simply rename it to "make" and remove the all. So it would be pull-cluster-api-make.